src/Raven.Server/Web/RequestHandlerContext.cs

```diff
@@ -11,5 +11,7 @@ public class RequestHandlerContext
         public RouteMatch RouteMatch;
         public DocumentDatabase Database;
         public bool CheckForChanges = true;
+
+        public string ClusterTransactionId => Database?.ClusterTransactionId;
```

Comment:
This looks like it is the cluster transaction of the _request_, but this is actually the global one?

Reply:
yes, this is the database cluster transaction id. (unique per database, not by request)